pkgs/applications/networking/mailreaders/neomutt/default.nix

```diff
@@ -1,6 +1,10 @@
 { stdenv, fetchFromGitHub, which, autoreconfHook, makeWrapper, writeScript,
 ncurses, perl , cyrus_sasl, gss, gpgme, kerberos, libidn, notmuch, openssl,
-lmdb, libxslt, docbook_xsl, docbook_xml_dtd_42, mime-types }:
+lmdb, libxslt, docbook_xsl, docbook_xml_dtd_42, mime-types
+, withSendmail ? false, nullmailer
```

Comment:
Instead of doing `,withFeature ? false, feature` you could just do: `feature ? null`.

Since `null` values are ignored within `buildInputs` you could also simplify the expression there.

Reply:
You mean like https://github.com/NixOS/nixpkgs/pull/32546/commits/bc889b9a6b0acc8888625c6ef78ccfd6c7e34ca1 ?